catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -265,6 +265,24 @@ public void cloneSpriteAndAddToStage(Sprite cloneMe) {
 			copy.look.setLookData(copy.getLookDataList().get(0));
 		}
 
+		copy.look.setPositionInUserInterfaceDimensionUnit(cloneMe.look.getXInUserInterfaceDimensionUnit(),
+				cloneMe.look.getYInUserInterfaceDimensionUnit());
+		copy.look.setTransparencyInUserInterfaceDimensionUnit(cloneMe.look.getTransparencyInUserInterfaceDimensionUnit());
+		copy.look.setColorInUserInterfaceDimensionUnit(cloneMe.look.getColorInUserInterfaceDimensionUnit());
+
+		int rotationMode = cloneMe.look.getRotationMode();
+		copy.look.setRotationMode(rotationMode);
+		if (rotationMode != Look.ROTATION_STYLE_LEFT_RIGHT_ONLY && copy.look.isFlipped()) {
+			copy.look.getLookData().getTextureRegion().flip(true, false);
+			copy.look.setFlipped(false);
+		}
+		boolean orientedLeft = copy.look.getDirectionInUserInterfaceDimensionUnit() < 0;
+		if (rotationMode == Look.ROTATION_STYLE_LEFT_RIGHT_ONLY && orientedLeft) {
+			copy.look.getLookData().getTextureRegion().flip(true, false);
+			copy.look.setFlipped(true);
+		}
+		copy.look.setDirectionInUserInterfaceDimensionUnit(cloneMe.look.getDirectionInUserInterfaceDimensionUnit());
+
```

Comment:
I like that idea! :)  Unfortunately, the clone calls [Sprite.resetSprite()](https://github.com/Catrobat/Catroid/blob/b5a9fc8323373031b4d561cba3d69de77d094319/catroid/src/main/java/org/catrobat/catroid/content/Sprite.java#L172) later on, which overwrites the look again. Do you see any reason why this method should be called at all after a sprite is cloned (I'm not sure about the physics stuff...)?